tests/functional/ancestral/cram/invalid-args.t

```diff
@@ -0,0 +1,68 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Input FASTA + VCF output is not possible
+
+  $ ${AUGUR} ancestral \
+  >  --tree $TESTDIR/../data/tree.nwk \
+  >  --alignment $TESTDIR/../data/aligned.fasta \
+  >  --output-vcf "$CRAMTMP/$TESTFILE/output.vcf" > /dev/null
```

Comment:
```suggestion
  >  --output-vcf output.vcf > /dev/null
```

`$CRAMTMP/$TESTFILE` isn't necessary since it's the default working directory for Cram tests. [Noted in DEV_DOCS](https://github.com/nextstrain/augur/blob/a5297310e8b53ccdfff625ded889b89be98ed2a8/docs/contribute/DEV_DOCS.md?plain=1#L113).

Reply:
Changed for this PR. I'll also update my subsequent PRs to use this.

Reply:
Sorry, I didn't realize that this test uses the setup script which changes directory away from the initial working directory. I think what you had previously was fine, but this is a small thing that I'd consider non-blocking as long as it works.

Reply:
Oh right! Sorry about all this. Confusion should go away after #1176 is done - it's not difficult, just a bit tedious, so I'd like to prioritize that work to be done sooner than later.